### PR TITLE
fix: update Angular experimental Workers + Assets template

### PR DESCRIPTION
**Pull request by petebacondarwin**
Fixes #0000

fix: update Angular experimental Workers + Assets template

---

- Tests
  - [ ] TODO (before merge)
  - [x] Tests included
  - [ ] Tests not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required
  - [x] Not required because: C3 e2e tests run automatically
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Documentation not necessary because: patch fix

### CODE REVIEW

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts [hunk 1]
```diff
@@ -74,6 +189,80 @@ function getFrameworkTests(opts: {
 				},
 				unsupportedOSs: ["win32"],
 			},
+			nuxt: {
+				quarantine: true,
```
Comment (emily-shen) [inline packages/create-cloudflare/e2e-tests/frameworks.test.ts:193]: just double checking, this is still supposed to quarantined?

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts [hunk 2]
```diff
@@ -74,6 +189,80 @@ function getFrameworkTests(opts: {
...
+						input: [keys.enter],
+					},
+					{
+						matcher:
+							/Which package manager do you want to install dependencies with/,
+						input: [keys.enter],
+					},
+				],
+				testCommitMessage: true,
+				unsupportedOSs: ["win32"],
```
Comment (emily-shen) [inline packages/create-cloudflare/e2e-tests/frameworks.test.ts:255]: just noticed no windows tests actually end up running 🤣 only gatsby supports windows, but we run pnpm for the windows test and gatsby doesn't support that. not sure if worth changing that the windows PM to npm or something.

**Review by emily-shen — APPROVED** [review]

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/create-cloudflare/e2e-tests/frameworks.test.ts:193]: It fails in this PR - but then passes in after the Nuxt PR 😄

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts (hunk 2 above)
Comment (petebacondarwin) [reply packages/create-cloudflare/e2e-tests/frameworks.test.ts:255]: We should tidy that up as a follow up

Comment (petebacondarwin) [reply packages/create-cloudflare/e2e-tests/frameworks.test.ts:255]: https://github.com/cloudflare/workers-sdk/issues/7412

**Pull request merged**